hack/README.md

```diff
@@ -0,0 +1,22 @@
+# hack
+
+This directory contains scripts that either generate the data maintained by OTA in this repo or use the data to display information about OpenShift update graph.
+
+
+* [exposure-length.sh](../hack/exposure-length.sh): It lists risk declaration for some or all risks with `fixedIn` available.
```

Comment:
nit: `../hack/` here and for the other links in this doc seem to be adding an unnecessary amount of indirection.  Can we use:

```Markdown
[`exposure-length.sh`](exposure-length.sh): ...
```

and similar?

Reply:
The paths in my mind were completely broken.